core/tests/protractor_desktop/editorFeatures.js

```diff
@@ -692,7 +692,7 @@ describe('Issues visualization', function() {
     workflow.publishExploration();
 
     adminPage.editConfigProperty(
-      'The set of exploration IDs for recording issues and playthroughs',
+      'The set of exploration IDs for recording playthrough issues',
```

Comment:
Just a suggestion on the wording.
```suggestion
      'The list of exploration IDs for recording playthrough issues',
```

Reply:
Hmm I actually prefer it this way. List just happens to be how we "implement" the set (for lack of a better data type).

Reply:
Sounds good thanks!